src/stdio.jl

```diff
@@ -32,20 +32,41 @@ end
 const bufs = Dict{String,IOBuffer}()
 const stream_interval = 0.1
 const max_bytes = 10*1024
+# max output per code cell is 512 kb
+const max_output_per_request = 1 << 19
```

Comment:
Either way we need a nice function for changing it so it shouldn't matter too much whether it's a global or a `Ref`. I ended up going with the latter.